cookbooks/steinberger/steinberger.prm

```diff
@@ -0,0 +1,224 @@
+  set Model name = spherical shell
+
+  subsection Spherical shell
+    set Inner radius  = 3481000
+    set Opening angle = 90
+    set Outer radius  = 6371000
+    set Phi periodic = true
+  end
+end
+
+# Both the top and bottom boundaries allow for free slip.
+subsection Boundary velocity model
+  set Tangential velocity boundary indicators = top, bottom
+end
+
+# Because the model has periodic side boundary conditions and free
+# slip at top and bottom, there is a rotational nullspace. We fix this
+# by setting the net rotation to zero.
+subsection Nullspace removal
+  set Remove nullspace = net rotation
+end
+
+# We use the gravity model from PREM.
+subsection Gravity model
+  set Model name = ascii data
+end
+
+# The temperature is prescribed at the top and bottom boundaries.
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = top, bottom
+  set List of model names = spherical constant
+
+  subsection Spherical constant
+    set Inner temperature = 3773
+    set Outer temperature = 273
+  end
+end
+
+# The initial temperature model consists of an adiabatic profile,
+# thermal boundary layers at the surface and the core-mantle boundary,
+# and a small harmonic perturbation to initiate convection.
+subsection Initial temperature model
+  set List of model names = adiabatic, function
+
+  subsection Function
+    set Coordinate system   = spherical
+    set Variable names      = r, phi
+    set Function constants  = r0 = 3481000, r1 = 6371000
+    set Function expression = 30 * sin((r-r0)/(r1-r0)*pi)*sin(16*phi) + 20 * sin(2*(r-r0)/(r1-r0)*pi)*sin(12*phi)
+  end
+
+  subsection Adiabatic
+    set Age top boundary layer = 1e8
+    set Age bottom boundary layer = 1e8
+  end
+end
+
+# The model uses the Steinberger material model, which uses a viscosity
+# profile constrained by mineral physics and the geoid, and which can read
+# in material properties from a look-up table.
+subsection Material model
+  set Model name = Steinberger
+
+  subsection Steinberger model
+
+    # This data directory contains the files for the viscosity profile,
+    # the lateral viscosity variations due to temperature, and all
+    # material files containing look-up tables computed by mineral physics
+    # software providing density, thermal expansivity and specific heat.
+    set Data directory                                = $ASPECT_SOURCE_DIR/data/material-model/steinberger/
+    set Lateral viscosity file name                   = temp-viscosity-prefactor.txt
+    set Material file names                           = pyr-ringwood88.txt
+    set Radial viscosity file name                    = radial-visc-simple.txt
+
+    # Because of the resolution in this cookbook we limit the overall viscosity
+    # between 1e20 Pa s and 5e23 Pa s and the lateral viscosity variations to
+    # three orders of magnitude in both directions (for a total of six orders of
+    # magnitude. This allows the features of the flow field to be resolved.
```

Comment:
I dont think this is how it works. Should `Maximum viscosity` and `Minimum viscosity` put a hard limit on viscosity independent of temperature variations?